### ISSUE TITLE
HyVideoTextEncode - All input tensors need to be on the same GPU, but found some tensors to not be on a GPU: [(torch.Size([1, 8388608]), device(type='cpu')), (torch.Size([262144]), device(type='cpu')), (torch.Size([4096, 4096]), device(type='cpu'))]

### ISSUE THREAD

**Issue by chaoscode** [kijai/ComfyUI-HunyuanVideoWrapper#130]
I don't know why this is an issue but it's detecting the cpu instead of the GPU on docker. The GPU is attached to the docker fine as I have mochi working on the same docker. Can anyone provide any guidance on how to fix this error?

**Comment by FlareP1** [comment]
Possiblye had a connected error see [Issue 80](https://github.com/kijai/ComfyUI-HunyuanVideoWrapper/issues/80), In my case the problem was that the `--lowvram` flag on launch was causing issues with memory allocation and forcing some tensors onto the CPU.

Are you using the `--lowvram` on the launch of comfyUI?  If so you could try to remove it?

**Comment by chaoscode** [comment]
I will look into it. I don't think it's a switch because I'm using the official docker from github on latest. Nothing special.

**Comment by chaoscode** [comment]
Yeah, I tried a non-lowvram workflow and it's just crashing the container. I think I'm just going to move on from this model. Thanks for the help.
